### PR TITLE
test

### PR DESCRIPTION
**Pull request by audi2014**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -17,4 +17,5 @@ jobs:
         run: |
         echo github.event.review.state
         exit 1
-      - uses: actions/checkout@v3
\ No newline at end of file
+      - uses: actions/checkout@v3
```
Comment (audi2014) [inline .github/workflows/main.yml:26]: comment

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -15,6 +15,7 @@ jobs:
           github.event_name == 'pull_request_review'
           && github.event.review.state != 'approved'
         run: |
-        echo github.event.review.state
-        exit 1
-      - uses: actions/checkout@v3
\ No newline at end of file
+          echo github.event.review.state
```
Comment (audi2014) [inline .github/workflows/main.yml]: tabs

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -15,6 +15,7 @@ jobs:
           github.event_name == 'pull_request_review'
           && github.event.review.state != 'approved'
         run: |
-        echo github.event.review.state
-        exit 1
-      - uses: actions/checkout@v3
\ No newline at end of file
+          echo ${{ github.event.review.state }}
```
Comment (audi2014) [inline .github/workflows/main.yml]: ok

File: .github/workflows/main.yml [hunk 4]
```diff
@@ -8,13 +8,15 @@ concurrency:
   cancel-in-progress: true
 jobs:
   npm-install:
+    if: github.event.review.state == 'approved'
```
Comment (audi2014) [inline .github/workflows/main.yml]: wow

Comment (audi2014) [reply .github/workflows/main.yml]: naswer

File: .github/workflows/main.yml [hunk 5]
```diff
@@ -8,13 +8,15 @@ concurrency:
   cancel-in-progress: true
 jobs:
   npm-install:
```
Comment (audi2014) [inline .github/workflows/main.yml]: one more comment

File: .github/workflows/main.yml [hunk 6]
```diff
@@ -8,13 +8,15 @@ concurrency:
   cancel-in-progress: true
```
Comment (audi2014) [inline .github/workflows/main.yml:9]: c

Comment (artur-perepelytsia-apiko) [inline .github/workflows/main.yml:9]: comment

**Review by artur-perepelytsia-apiko — DISMISSED** [review]

File: .github/workflows/main.yml [hunk 7]
```diff
@@ -8,13 +8,15 @@ concurrency:
...
   npm-install:
+    if: github.event.review.state == 'approved'
     runs-on: ubuntu-latest
     steps:
       - name: Reject if not approved
         if: |
           github.event_name == 'pull_request_review'
           && github.event.review.state != 'approved'
         run: |
-        echo github.event.review.state
```
Comment (artur-perepelytsia-apiko) [inline .github/workflows/main.yml:18]: comment

**Review by artur-perepelytsia-apiko — DISMISSED** [review]

**Review by artur-perepelytsia-apiko — COMMENTED** [review]
test

**Review by artur-perepelytsia-apiko — COMMENTED** [review]
trigger ci

**Review by artur-perepelytsia-apiko — COMMENTED** [review]
trigger ci

**Review by artur-perepelytsia-apiko — COMMENTED** [review]
trigger ci

**Review by artur-perepelytsia-apiko — COMMENTED** [review]
trigger ci

**Review by artur-perepelytsia-apiko — APPROVED** [review]

**Review by audi2014 — COMMENTED** [review]
trigger ci

**Pull request merged**